src/helpers/data.ts

```diff
@@ -115,12 +115,24 @@ export const BaseArmorSets: ArmorSet[] = [
   }, {
     name: "Navigator's Helm",
     helm: {armor: 16, weight: 7.6, bleed: -1, burn: -1, overload: 2, blight: -1, corrode: 3},
+  }, {
+    name: "Nanoplated",
+    helm: {armor: 12.4, weight: 5, bleed: 2, burn: 1, overload: 0, blight: 2, corrode: 0},
+    chest: {armor: 29.6, weight: 12, bleed: 4, burn: 2, overload: 0, blight: 4, corrode: 0},
+    boot: {armor: 7.4, weight: 3, bleed: 1, burn: 1, overload: 0, blight: 1, corrode: 0},
```

Comment:
the values for boots seems to be 14.8 armor, 6 weight, 3/2/0/2/0 resistances. This looks to be the values for the gloves instead